src/modules/Header/features/LangSwitch/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import styles from './style.module.css';
+
+export const LangSwitch = () => {
+  return (
+    <select className={styles.select}>
+      <option value="Eng">Eng</option>
```

Comment:
value should be from small letter and have a special language codes :

English: "en-US", 
Polish: "pl-PL",
Ukrainian: "uk-UK"